x-pack/plugin/src/yamlRestTest/resources/rest-api-spec/test/esql/60_usage.yml

```diff
@@ -30,7 +30,7 @@ setup:
         - method: POST
           path: /_query
           parameters: []
-          capabilities: [ snapshot_test_for_telemetry ]
+          capabilities: [ fn_bit_length ]
```

Comment:
@timgrein apologies, I forgot that when I added the two different capabilities I did it with a twist:

```
SNAPSHOT_TEST_FOR_TELEMETRY(Build.current().isSnapshot())
NON_SNAPSHOT_TEST_FOR_TELEMETRY(Build.current().isSnapshot() == false)
```

I think these two capabilities must **always** be placed in those two tests, together with the new capability: `capabilities: [ snapshot_test_for_telemetry, fn_bit_length ]`